lib/api.js

```diff
@@ -4,9 +4,9 @@ var url = require('url');
 var request = require('request');
 var Q = require('q');
 var xml2js = require('xml2js');
+var credentials = require('plex-api-credentials');
```

Comment:
Yupp, true enough. Usually shy away from require inside methods as they are
synchronous under the hood, but thats really not an issue for this kind of
module.
On tor. 23. jul. 2015 at 00.04 Greg Laabs <notifications@github.com> wrote:

> In lib/api.js
> <https://github.com/phillipj/node-plex-api/pull/13#discussion_r35269959>:
>
> > @@ -4,9 +4,9 @@ var url = require('url');
> >  var request = require('request');
> >  var Q = require('q');
> >  var xml2js = require('xml2js');
> > +var credentials = require('plex-api-credentials');
>
> Might want to move this inside of _credentialsAuthenticator() ?
>
> It's not common practice to put a module require() inside a method, but
> otherwise we'll be loading in the full module (small as it is) even in
> cases where it won't ever be used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/phillipj/node-plex-api/pull/13/files#r35269959>.
>